scripts/update_node.py

```diff
@@ -45,7 +45,7 @@
 
     upload_url = upload_base + filename
     print('Uploading: ' + upload_url)
-    cmd = ['gsutil', 'cp', '-n', filename, upload_url]
+    cmd = ['gsutil.py', 'cp', '-n', filename, upload_url]
```

Comment:
My gsutil from google-cloud-sdk is shell script with no extension.   Where is yours coming from?

Reply:
Oops, that was supposed to be just a local change. gsutil.py is included in depot_tools, so I was just using that.